README.md

```diff
@@ -66,15 +66,71 @@ https://github.com/btcsuite/btcwallet/releases
 
 ### Windows/Linux/BSD/POSIX - Build from source
 
-- If necessary, install Go according to the installation instructions
-  here: http://golang.org/doc/install.  It is recommended to add
-  `$GOPATH/bin` to your `PATH` at this point.
+Building or updating from source requires the following build dependencies:
+
+- **Go 1.5 or 1.6**
+
+  Installation instructions can be found here: http://golang.org/doc/install.
+  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
+
+- **Glide**
+
+  Glide is used to manage project dependencies and provide reproducable builds.
+  To install:
+
+  `go get -u github.com/Masterminds/glide`
+
+Unfortunately, the use of `glide` prevents a handy tool such as `go get` from
+automatically downloading, building, and installing the source in a single
+command.  Instead, the latest project and dependency sources must be first
+obtained manually with `git` and `glide`, and then `go` is used to build and
+install the project.
+
+**Getting the source**:
+
+For a first time installation, the project and dependency sources can be
+obtained manually with `git` and `glide` (create directories as needed):
+
+```
+git clone https://github.com/btcsuite/btcwallet $GOPATH/src/github.com/btcsuite/btcwallet
+cd $GOPATH/src/github.com/btcsuite/btcwallet
+glide install
+```
+
+To update an existing source tree, pull the latest changes and install the
+matching dependencies:
+
+```
+cd $GOPATH/src/github.com/btcsuite/btcwallet
+git pull
+glide install
+```
+
+**Building/Installing**:
+
+The `go` tool is used to build or install (to `GOPATH`) the project.  Some
+example build instructions are provided below (all must run from the `btcwallet`
+project directory).
+
+To build a `btcwallet` executable and place it in the current directory:
+
+```
+go build
+```
+
+To build a `btcwallet` executable and install it to `$GOPATH/bin/btcwallet`:
+
+```
+go get
```

Comment:
Correct.  That is why I was suggesting `install`, because the entire point of switching over to glide is to ensure reproducible builds.  I would rather have it fail to build/install if there are any dependencies missing which would mean that the lockfile isn't specifying them properly.